packages/components/src/truncate/types.ts

```diff
@@ -17,19 +29,22 @@ export interface Props {
 	 *
 	 * @default 'auto'
 	 */
-	ellipsizeMode?: TruncateEllisizeMode;
+	ellipsizeMode?: TruncateEllipsizeMode;
 	/**
 	 * Determines the max characters when `truncate` is set.
 	 *
 	 * @default 0
 	 */
 	limit?: number;
 	/**
-	 * Clamps the text content to the specified `numberOfLines`, adding the `ellipsis` at the end.
+	 * Clamps the text content to the specified `numberOfLines`, adding the
+	 * `ellipsis` at the end.
+	 *
```

Comment:
During my tests, the actual behaviour of the component differs from this description — in particular, the "ellipsis" shown to truncate the text seems to always be `...`, regardless of the value of the `ellipsis` prop.

Maybe we can update the description? Something like:

_Clamps the text content to the specified `numberOfLines`, adding an ellipsis at the end. Note: this feature ignores the value of the `ellipsis` prop and always displays the default `...` ellipsis._

Reply:
🥲 https://caniuse.com/?search=text-overflow%20string

Reply:
😞  I know!

Although the code has the `text-overflow: ellipsis` rule [hardcoded](https://github.com/WordPress/gutenberg/blob/13c4d3ae0afcb8fce38fbe9ce5babbbdd1bd149e/packages/components/src/truncate/styles.ts#L9), so it seemed somewhat deliberate (but not documented)